Algorithms/Other/KochSnowflake.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Numerics;
+
+namespace Algorithms.Other
+{
+    /// <summary>
+    /// The Koch snowflake is a fractal curve and one of the earliest fractals to
+    /// have been described. The Koch snowflake can be built up iteratively, in a
+    /// sequence of stages. The first stage is an equilateral triangle, and each
+    /// successive stage is formed by adding outward bends to each side of the
+    /// previous stage, making smaller equilateral triangles.
+    /// This can be achieved through the following steps for each line:
+    ///     1. divide the line segment into three segments of equal length.
+    ///     2. draw an equilateral triangle that has the middle segment from step 1
+    ///     as its base and points outward.
+    ///     3. remove the line segment that is the base of the triangle from step 2.
+    /// (description adapted from https://en.wikipedia.org/wiki/Koch_snowflake )
+    /// (for a more detailed explanation and an implementation in the
+    /// Processing language, see  https://natureofcode.com/book/chapter-8-fractals/
+    /// #84-the-koch-curve-and-the-arraylist-technique ).
+    /// </summary>
+    public static class KochSnowflake

```

Comment:
Could you make only one GetBitmap method public with only width parameter (it's a square) and number of steps parameter? I'd like to be able to draw this snowflake easily and it's hard to do with current parameter initialVectors.

Reply:
That's a good idea. This way it's more user-friendly.